BEP82.md

```diff
@@ -0,0 +1,72 @@
+# BEP-82: Token Owner Changes
+
+## 1. Summary
+This BEP describes the changes related to the token owner who issued a token on Binance Chain.
+
+## 2. Abstract
+Currently, many token-related transactions, such as token listing, minting, burning, etc., can only be proposed by the token owner who can not be changed once the token is issued on Binance Chain.
+
+This BEP proposes the solutions to provide more convenience and flexibility for these transactions.
+
+## 3. Status
+draft
+
+## 4. Motivation
+To reduce the transaction's dependence on the related token’s owner and make the ownership of token changeable, we introduce those solutions as following:
+- Providing an entrance to transfer the ownership of a specific token.
+- Removing the owner verification when handling some token related transactions.
+
+## 5. Specification
+### 5.1 Owner Transfer
+**OwnerTransfer** transaction can transfer ownership of a specific token to another address, and only the original owner has the permission to send this transaction.
```

Comment:
it is better named as `transferOwnership`

Reply:
renamed